README.md

```diff
@@ -84,6 +84,18 @@ You can use Base64 parameters for passing _small_ files to downstream builds:
 build job: 'downstream', parameters: [base64File(name: 'file', base64: Base64.encoder.encodeToString('hello'.bytes)))]
 ```
 
+## Usage with REST API
+
+You can pass file parameters to the REST API:
```

Comment:
`buildWithParameters`, especially with file uploads, seems to strain the definition of https://en.wikipedia.org/wiki/Representational_state_transfer#Architectural_constraints. https://www.jenkins.io/doc/book/using/remote-access-api/ does refer to a _REST-like style_ though actually this refers to the read-only “export” system https://javadoc.jenkins.io/component/stapler/org/kohsuke/stapler/export/package-summary.html + https://javadoc.jenkins.io/hudson/model/Api.html. Nitpicking but maybe the wording can be relaxed to just say a “remote” or “HTTP” API?

Reply:
I can certainly do that. Although since Jenkins itself seems to refer to `buildWithParameters` as part of their REST API, it will probably be a good idea to at least have that name in a parenthetical note to help people to land in a successful conclusion